Rating/Rating.css

```diff
@@ -0,0 +1,76 @@
+:root{
+    --special-color:black;
+    --border: 1px solid black;
+    --special-padding:1rem;
+    --special-margin-top:1rem;
+    --font-size:1.5rem;
+}
+.code{
+    margin-top: 2rem;
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+    background-color:rgb(139, 185, 179);
```

Comment:
You can create a CSS variable for the color if it is being used in other places as well